utils/searx.sh

```diff
@@ -333,6 +333,7 @@ git pull
 pip install -U pip
 pip install -U setuptools
 pip install -U wheel
+pip install -U pyyaml
```

Comment:
`pip install -U -e .` should install pyyaml since requirements.txt contains a reference to pyyaml:
https://github.com/searx/searx/blob/5a511f0d620038b8e94c581bcfd3c987082b9414/requirements.txt#L10

or am I missing something ?

Reply:
It is `pip install -r`  ...

https://github.com/searx/searx/blob/5a511f0d620038b8e94c581bcfd3c987082b9414/utils/makefile.python#L125

which installs packages directly from the requirements file.  On the opposite   `pip install -U -e .` will _execute_ the setup function ...

https://github.com/searx/searx/blob/5a511f0d620038b8e94c581bcfd3c987082b9414/setup.py#L22-L23

from the `setup.py` file.  For this pip (or better say [setuptools](https://setuptools.readthedocs.io/en/latest/) ) will import the `setup.py` file.

Reply:
Some additional hints: pip is the command line to get access to a package management and infrastructure.  At the end the package itself is installed by setuptools:

https://github.com/searx/searx/blob/5a511f0d620038b8e94c581bcfd3c987082b9414/setup.py#L4

https://github.com/searx/searx/blob/5a511f0d620038b8e94c581bcfd3c987082b9414/setup.py#L22

If you have all requirements installed, you can also call:

    pyhon setup.py install        # use --help to see options

to install the package.